ig/input/fsh/PopulationRecommendation.fsh

```diff
@@ -3,7 +3,8 @@ Parent: Basic
 Id: vp-population-recommendation
 Title: "PopulationRecommendation"
 Description: ""
```

Comment:
We forgot to model the Reference to TargetDisease.

Reply:
I have added a `targetDisease` extension (`1..1 MS`) with an ICD-10 code of the disease to immunise against.

Reply:
Great!